compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -2280,6 +2280,11 @@ public void visit(BLangForever foreverStatement) {
 
     @Override
     public void visit(BLangConstant constant) {
+        Name constantName = names.fromIdNode(constant.name);
+        if (constantName == Names.IGNORE) {
+            dlog.error(constant.name.pos, DiagnosticCode.CANNOT_DEFINE_CONSTANT_WITH_IDENTIFIER, constant.name);
+            return;
```

Comment:
Do we need the return here? We can allow the next check also right?

Reply:
Fixed.